src/actions/write/decreaseDelegateStake.ts

```diff
@@ -25,7 +25,7 @@ export class DecreaseDelegateStake {
     }
     const { target, qty } = input;
     this.target = target;
-    this.qty = new IOToken(qty);
+    this.qty = new mIOToken(qty); // round to avoid bad user input errors
```

Comment:
would this comment be better as a tsdoc of `mIOToken`? e.g `/** Rounds (Up/Down) input ... */`

Reply:
it's actually an outdated comment - the validation requires full integers so no rounding happens

Reply:
[60a3aa7](https://github.com/ar-io/testnet-contract/pull/214/commits/60a3aa700c50a8887bbd557315c1d5b263ecc166)